src/tcp.facility.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const _isNil = require('lodash/isNil')
```

Comment:
I'm trying to avoid moria importing `lodash`, lodash is discontinued, so we need to slowly transition away, so new projects should avoid it